doc/pages/About.md

```diff
@@ -1,60 +1,62 @@
 # opam
 
-## A little bit of History
+## A Little Bit of History
 
 ### opam
 
-The [first specification draft of OPAM](https://github.com/ocaml/opam/blob/30598a59c98554057ce2beda80f0d31474b94150/specs/roadmap.pdf?raw=true)
-was written at the end of Jan 2012 by Thomas Gazagnaire from OCamlPro. The specification was reviewed by Fabrice Le Fessant (OCamlPro/INRIA), Yaron Minsky (Jane Street) -- who funded the project, and Anil Madhavapeddy (University of Cambridge) -- who needed a source-based package manager to manage libraries emerging from the Mirage OS project.
+Thomas Gazagnaire of Tarides wrote the [first specification draft of opam](https://github.com/ocaml/opam/blob/30598a59c98554057ce2beda80f0d31474b94150/specs/roadmap.pdf?raw=true) at the end of Jan 2012, when he was still working with OCamlPro. The specification was reviewed by Fabrice Le Fessant (OCamlPro/INRIA), Yaron Minsky of Jane Street (who funded the project), and Anil Madhavapeddy (University of Cambridge) who needed a source-based package manager to manage libraries emerging from the MirageOS project.
 
-At about the same time, Frederic Tuong, an intern from Inria, funded by the DORM research grant in collaboration with OCamlPro and IRILL, started to implement [the first version of OPAM](https://github.com/ocaml/opam/commits/master?page=112)
-(called `ocp-get` at the time) at the end of Feb 2012. He also started to create the
-[first OPAM packages](https://github.com/ocaml/opam-repository/commits/master?page=200) 
+At about the same time, Frederic Tuong, an intern from Inria started to implement [the first version of opam](https://github.com/ocaml/opam/commits/master?page=112)
+(called `ocp-get` at the time) at the end of Feb 2012. The DORM research grant funded this project in collaboration with OCamlPro and IRILL. Frederic also started to create the
+[first opam packages](https://github.com/ocaml/opam-repository/commits/master?page=200) 
```

Comment:
This link is outdated. This is now showing commits from last year

Reply:
Do you know what link it should be?